clients/tests/CMakeLists.txt

```diff
@@ -141,19 +141,16 @@ foreach( target ${TEST_TARGETS} )
   endif()
 
   if( BUILD_CLIENTS_TESTS_OPENMP )
+    find_package(OpenMP REQUIRED)
     if( BUILD_WITH_LIB STREQUAL "CUDA" )
       message( STATUS "OpenMP is not supported on CUDA, building tests without it" )
     else()
       target_compile_options( ${target} PRIVATE -DBUILD_CLIENTS_TESTS_OPENMP )
       if(NOT (CMAKE_CXX_COMPILER MATCHES ".*hipcc$" OR CMAKE_CXX_COMPILER MATCHES ".*clang\\+\\+"))
-        target_compile_options( ${target} PRIVATE -fopenmp )
-        target_link_libraries( ${target} PRIVATE -fopenmp -L${HIP_CLANG_ROOT}/lib -Wl,-rpath=${HIP_CLANG_ROOT}/lib )
+        target_link_libraries( ${target} PRIVATE OpenMP::OpenMP_CXX -L${HIP_CLANG_ROOT}/lib -Wl,-rpath=${HIP_CLANG_ROOT}/lib )
```

Comment:
I suspect we don't need `-L${HIP_CLANG_ROOT}/lib -Wl,-rpath=${HIP_CLANG_ROOT}/lib` anymore, if CMake is told properly where OpenMP is.  Would you be able to try removing it?

Reply:
Good call!  I've removed them; everything seems fine.